frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -678,6 +681,20 @@ const VariantsTable: FC<VariantsTableProps> = ({
     notIncludedVariants && notIncludedVariants.has(variant.id) ? 1 : 0
   );
 
+  const listRef = useRef<FixedSizeList | null>(null);
```

Comment:
Nice on working this out. 

I personally find dealing with Reacts refs a bit hard to wrap my head around sometimes.